app/components/OceanInterface/OceanInterface.tsx

```diff
@@ -107,36 +142,33 @@ function TransactionError ({ txid }: { txid: string | undefined }): JSX.Element
           style={tailwind('text-sm font-bold')}
         >{`${translate('screens/OceanInterface', 'An error has occurred')}`}
         </Text>
-        {
```

Comment:
I think we shouldn't remove this TxID. The idea of this Transaction error is to display if there's any JS errors or if there's an error related to the transaction.

Unless there's no case where a txid is broadcasted and an error will occur. We can remove this

Reply:
As now signing itself as a promise can fail, so txid is not always available. But make sense to display it if we have one. Will include it in the message crafting if signing did went through.